salesagility/SuiteCRM
# Feature/gdpr opt in status

Notes: 
- this pull request had a conflict with #5010  (displayEmailAddressOptInField.tpl)
- this pull request had a conflict with #5020

## Description



I have made some improvements to the opt in status to make it look a little better. 

When the email settings are configured to "opt-in" and an email address has been "opted-in", a "green" tick should be displayed next to the email address instead of a "grey" tick.

When the email settings are configured to "confirmed-opt-in" and an email address has been "confirmed-opt-in". The two ticks should be displayed next to the email address and the ticks have no spacing in between.

When user edits a contact/account they can control the opt in using the opt-in check box.

## Motivation and Context

- GDPR

## How To Test This

- run **composer update** (yay namespaces)
- Repair and Rebuild
- Configure Emails Settings
- Go to contacts and visual check the indicators next to the email addresses.
- Edit a contact and see how the opted in checkbox effects the indicator in the detail view.

Also, please test that the functionality in #5010 and #5020

## Types of changes


### Final checklist
